### PR TITLE
Remove optional dependencies for legacy Python versions

### PR DESCRIPTION
**Pull request by michaelmanganiello-eb**
Remove `unittest2` and `simplejson` dependencies, which were needed for
Python versions prior to 2.7, so we don't need to keep them.

### CODE REVIEW

File: txlib/utils/imports.py [hunk 1]
```diff
@@ -1,18 +0,0 @@
-# -*- coding: utf-8 -*-
-
-"""
-Define shortcuts to various imports needed.
-"""
-
-import sys
-if sys.version_info < (2, 6):
```
Comment (nickwilliams-eventbrite) [inline txlib/utils/imports.py]: :LOLFACE:

File: txlib/utils/imports.py [hunk 2]
```diff
@@ -1,18 +0,0 @@
...
-    import simplejson as json
-else:
-    import json
-
-
-import sys
-if sys.version_info < (2, 7):
```
Comment (nickwilliams-eventbrite) [inline txlib/utils/imports.py]: :LOLFACE:

File: setup.py [hunk 3]
```diff
@@ -2,20 +2,12 @@
...
-try:
-    import unittest2
-except ImportError:
-    install_requires.append('unittest2')
+install_requires = []
 
 setup(
     name="txlib",
-    version="0.1+eventbrite",
+    version="0.1.1+eventbrite",
```
Comment (nickwilliams-eventbrite) [inline setup.py]: Given what it is and how old it is, etc., I won't raise a stink, but are you sure you should do this as a patch and not a minor? I'm fine with it, just want to challenge you to think it over and be sure this is right.

Comment (michaelmanganiello-eb) [reply setup.py]: That's a valid thought. My approach to this PR was that this library requires `unittest2`, and is forcing projects to install it. Considering that, I implemented it as a bugfix, with a related patch release.

First solution was to change this requirement to be in `tests_require` instead, but as Python 2.6 has been deprecated for so long, none of this code would be needed, and we're not considering it to be a breaking change.

As you said, this versioning schema is extremely important, considering how often we make changes to this project.

**Pull request merged**